src/lib/target/inc/osn_maptm.h

```diff
@@ -0,0 +1,54 @@
+/*
+* Copyright (c) 2020, Sagemcom.
+* All rights reserved.
+*
+* The information and source code contained herein is the exclusive property of
+* Sagemcom and may not be disclosed, examined, or reproduced in whole or in part
+* without explicit written authorization of Sagemcom.
+*/
```

Comment:
This is not an open source license. It should be replaced with a BSD-3 license.
Example: https://github.com/plume-design/opensync/blob/osync_2.0.0/src/lib/osn/inc/osn_fw.h
See also: https://www.opensync.io/contributing-to-opensync (_Licensing of Contributions_)

(applies to all files/occurrences)

Reply:
OK,it's planning with the code style review